tests/test_pot_iterations.py

```diff
@@ -99,37 +92,33 @@ def test_calculate_ip_iters(self):
         with raises(ValueError):
             # required_iters too high
             calculate_ip_iters(
-                test_constants.NUM_SPS_SUB_SLOT,
-                test_constants.NUM_SP_INTERVALS_EXTRA,
+                test_constants,
                 ssi,
-                sp_interval_iters,
```

Comment:
these were failing. I think it was a mistake to pass in `sp_interval_iters` as `signage_point_index`. This value doesn't fit in a `u8`

Reply:
it also doesn't make much sense, these are unrelated